main.nginx.conf

```diff
@@ -95,7 +95,10 @@ http {
     }
 
     location = /client-config.json {
-      include ./common-headers.nginx.conf;
```

Comment:
We don't call `add_header` in this `location`, so I don't think we need to include this file. Locally, I can see that the expected headers are coming through.

Reply:
Should this change be a separate PR?  It seems to be unrelated to the aim of this PR.

Reply:
I would keep this line, as it doesn't do any harm, and keeps this file closer to central's `files/nginx/odk.conf.template`, and therefore easier to compare.

We could go further in mirroring central's `files/nginx/odk.conf.template` by adding:

```nginx
add_header Cache-Control no-cache;
```